pycbc/__init__.py

```diff
@@ -96,12 +95,9 @@ def sig_handler(signum, frame):
 
     signal.signal(signal.SIGUSR1, sig_handler)
 
-    if not verbose:
-        initial_level = logging.WARN
-    elif int(verbose) == 1:
-        initial_level = logging.INFO
-    else:
-        initial_level = int(verbose)
+    initial_level = logging.WARN
```

Comment:
Seems OK, except that the code is now a bit cryptic. Suggest to add a comment making people consult https://docs.python.org/3/library/logging.html#levels to explain what is happening. 
Also, `logging.WARN` seems to be deprecated in favor of `logging.WARNING` so can we have that.